src/script.ts

```diff
@@ -0,0 +1,694 @@
+    } else if (code === 11 || code === 21) {
+      document.body.className = document.body.className.replace(/\btheme-\S+\b/g, '').trim()
+      document.body.classList.add("theme-stormy");
+      themeKey = 'stormy';
+    } else if (code === 15 || code === 16 || code === 17 || code === 25 || code === 26 || code === 27) {
+      document.body.className = document.body.className.replace(/\btheme-\S+\b/g, '').trim()
+      document.body.classList.add("theme-snowy");
+      themeKey = 'snowy';
+      divConditionStyle.innerHTML = `
+        <div class="snowfall">
+          <div class="snow"></div>
+          <div class="snow"></div>
+          <div class="snow"></div>
+          <div class="snow"></div>
+          <div class="snow"></div>
+        </div>
+      `;
+    }
+
+    const theme = THEMES[themeKey]
+    const meta = await metaBox(result, city);
+    
+    wrapper?.appendChild(meta);
+    wrapper?.appendChild(conditionBox(theme.h1Text.replace(cities[0]!.name, city.name), theme.iconSvg));
+    wrapper?.appendChild(weatherWeekBox(result));
+    search();
+
+  } catch(error) {
+    console.log(`Error fetching: ${error}`);
+  }
+}
+
+// ! ||--------------------------------------------------------------------------------||
+// ! ||                        Show multiple cities desktop mode                       ||
+// ! ||--------------------------------------------------------------------------------||
+
+const toggleButton = document.createElement("button");
+toggleButton.textContent = "☰";
+toggleButton.className = "toggle-btn";
+document.body.prepend(toggleButton);
```

Comment:
What an easter egg! Love that you can compare. Any swedes wet dream!? A weather dashboard like this haha